docs/pt/docs/deployment/index.md

```diff
@@ -0,0 +1,7 @@
+# Implantação - Introdução
+
+A implantação de uma aplicação **FastAPI** é relativamente simples.
+
+Existem várias maneiras para fazer isso, dependendo do seu caso específico e das ferramentas que você utiliza.
```

Comment:
Eu acho que ficaria mais completo se fosse adicionado `caso de uso especifico`.
```suggestion
Existem várias maneiras para fazer isso, dependendo do seu caso de uso específico e das ferramentas que você utiliza.
```

Reply:
In a previous discussion, we decided to remove the `use case`, because it sounded redundant. `use case` is already `specific`, hence they can't be together we decided to stay with only specific

Reply:
Ok, that's a really good point that I didn't thinked of !!!